src/eduid_userdb/event.py

```diff
@@ -100,6 +102,13 @@ def __init__(
             # Just keep everything that is left as-is
             self._data.update(data)
 
+    @classmethod
+    def from_dict(cls: Type['Event'], data: Dict[str, Any], raise_on_unknown: bool = True) -> 'Event':
```

Comment:
import annotations and don't declare return type using a string

Reply:
:+1: